webpack.config.js

```diff
@@ -27,8 +15,10 @@ module.exports = {
           presets: ["@babel/preset-env", "@babel/preset-react"]
         },
       },
+      {
```

Comment:
So is the result of this CSS loader that CSS doesn't have to all load before page displays? Or is it for preloading images..? Either way, if it saves 1.2s then it is worth it. I wonder if a different type of minification works with it